### DISCUSSION TITLE
GPSR - pobieranie pojedynczych elementów

### DISCUSSION THREAD

**Discussion by tztztztz** [allegro/allegro-api#10530] [Suggestions]
Dzień dobry,

bardzo przydałyby się metody do pobierania pojedynczych elementów GPSR na podstawie w stylu:

https://api.{environment}/sale/responsible-persons/{id}

w tym momencie, kiedy chcielibyśmy pobrać jakiś element na podstawie id, musimy iterować po całej kolekcji

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Jeśli zdecydujemy się wprowadzić to rozwiązanie, poinformujemy w wątku.

**Comment by creativepl-php** [comment]
Zdecydowanie na tak. Filtrowanie powinno być przynajmniej po tradeName - jeśli w danym producencie istnieje fraza (`tradeName` LIKE '%fraza%') to powinno zwracać producentów z takim matchem.
